testing-tools/Dockerfile

```diff
@@ -43,6 +44,11 @@ RUN apt-get update && \
 COPY testing-tools/python /stackable/python
 COPY testing-tools/licenses /licenses
 
+ENV PATH=/stackable/keycloak/bin:$PATH
+# todo: do we need to maintain this via the mirror script?
```

Comment:
```suggestion
```

Reply:
Done in 951720f6fe42ee4a37414d9d164ce61888e93cd1